test/Chirp.InfrastructureTests/RepositoryTests/CheepRepositoryTest.cs

```diff
@@ -11,10 +11,18 @@ public class CheepRepositoryTest{
     private readonly CheepRepository CheepRepository;
     private readonly ChirpDbContext db;
 
+    private Author _author;
+
     public CheepRepositoryTest()
     {
         db = SqliteInMemoryBuilder.GetContext();
         CheepRepository = new CheepRepository(db);
+
+        _author = new Author()
```

Comment:
Nice to see that you have added the authorDTO to the constructor to reduce redundancy